### PR TITLE
Explain staging and development versions

### LINKED ISSUE #186: Document staging/dev (stable?) OBS
In the quickstart docs the `helm upgrade` is wrapped in Tabs for staging and dev.  The warning under dev implies dev is more bleeding edge ("unstable"/"experimental").

Nothing seems to define what staging/dev mean.

Looking at OBS I see two more points of confusion:
- There is also a stable
- dev (v1.2.4) is currently behind staging/stable (v.1.2.5)

Please document:
- What is staging
- What is dev
- Why is stable ignored in the tab options
  - If stable should be in tabs, when to use stable instead of staging
- When to use dev (current warning only includes reasons not to use it)

OBS pages I was looking at for version inputs:
- https://build.opensuse.org/package/show/isv:Rancher:Elemental:Stable/elemental-operator-crds-helm
- https://build.opensuse.org/package/show/isv:Rancher:Elemental:Staging/elemental-operator-crds-helm
- https://build.opensuse.org/package/show/isv:Rancher:Elemental:Dev/elemental-operator-crds-helm

### PR DESCRIPTION
**Pull request by kkaempf**
Fixes #186

### CODE REVIEW

File: versioned_docs/version-1.2/partials/_elemental-operator-install.md [hunk 1]
```diff
@@ -19,6 +27,15 @@ When upgrading from an elemental-operator release embedding the Elemental CRDs (
 You will need to upgrade the elemental-operator chart first, and only then install the elemental-operator-crds chart.
 :::
 
+### Non-stable installations
+
+Besides the Helm charts listed above, there are two other, non-stable
+versions available.
+
+* Staging - refers to the latest tagged release from Github. This is documented in the [Next](next/quickstart-ui) pages.
```
Comment (fgiudici) [inline versioned_docs/version-1.2/partials/_elemental-operator-install.md:35]: Same comment as above (just for tracking)

**Review by fgiudici — APPROVED** [review]
Looks good here!
Just a bit worried about stating the Next <-> Staging relationship as it happens we document stuff in the Development channel not yet landed in staging... users may find so staging missing features they expect to have there

File: docs/partials/_elemental-operator-install.md [hunk 2]
```diff
@@ -19,6 +27,15 @@ When upgrading from an elemental-operator release embedding the Elemental CRDs (
 You will need to upgrade the elemental-operator chart first, and only then install the elemental-operator-crds chart.
 :::
 
+### Non-stable installations
+
+Besides the Helm charts listed above, there are two other, non-stable
+versions available.
```
Comment (nunix) [inline docs/partials/_elemental-operator-install.md:33]:
```suggestion
Besides the Helm charts listed above, there are two other `non-stable`
versions available.
```

File: docs/partials/_elemental-operator-install.md [hunk 3]
```diff
@@ -19,6 +27,15 @@ When upgrading from an elemental-operator release embedding the Elemental CRDs (
...
+
+* Staging - refers to the latest tagged release from Github. This is documented in the [Next](next/quickstart-ui) pages.
```
Comment (nunix) [inline docs/partials/_elemental-operator-install.md:35]:
```suggestion
* **Staging:** refers to the latest tagged release from Github. This is documented in the [Next](next/quickstart-ui) pages.
```

File: docs/partials/_elemental-operator-install.md [hunk 4]
```diff
@@ -19,6 +27,15 @@ When upgrading from an elemental-operator release embedding the Elemental CRDs (
...
+
+* Development - refers to the 'tip of HEAD' from Github. This is the ongoing development version and changes constantly.
```
Comment (nunix) [inline docs/partials/_elemental-operator-install.md:37]:
```suggestion
* **Development:** refers to the 'tip of HEAD' from Github. This is the ongoing development version and changes constantly.
```

File: versioned_docs/version-1.2/partials/_elemental-operator-install.md (hunk 1 above)
Comment (nunix) [inline versioned_docs/version-1.2/partials/_elemental-operator-install.md:35]:
```suggestion
* **Staging:** refers to the latest tagged release from Github. This is documented in the [Next](next/quickstart-ui) pages.
```

**Review by nunix — COMMENTED** [review]
Few format changes

File: docs/partials/_elemental-operator-install.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (kkaempf) [reply docs/partials/_elemental-operator-install.md]: Hmm, how do we update the docs on a(n Elemental) release then ?
We can't just copy docs/* to versioned_docs 🤔

**Pull request merged**